ui/admin/app/abilities/auth-method.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import ModelAbility from 'api/abilities/model';
+
+export default class OverrideAuthMethodAbility extends ModelAbility {
+  /**
+   * These override abilities are temporary and just part of Phase 1
+   * of LDAP auth method implementation.
+   */
+
+  get canRead() {
+    return !this.model.isLDAP && super.canRead;
```

Comment:
That is a really good idea. An `isUnknown` getter in the model and a auth-method ability file in the addon level. So it would  be similar to other resources like [credential-library](https://github.com/hashicorp/boundary-ui/blob/main/addons/api/addon/abilities/credential-library.js). @DhariniJeeva thoughts on adding this change to this pr?

Reply:
I like the idea of adding `isUnknown` similar to the existing pattern! 👍